NeoMathEngine/src/CPU/CpuMathEngineBlas.cpp

```diff
@@ -643,8 +629,8 @@ void CCpuMathEngine::MultiplyMatrixByMatrix( int batchSize, const CConstFloatHan
 	CFloatHandle result = resultHandle;
 
 	for( int b = 0; b < batchSize; ++b ) {
-		multiplyMatrixByMatrix( first, firstHeight, firstWidth, firstWidth, second, secondWidth, secondWidth, result,
-			secondWidth, firstHeight * secondWidth );
+		multiplyMatrixByMatrix( GetRaw( first ), firstHeight, firstWidth, firstWidth, GetRaw( second ), 
```

Comment:
Better change `first`, `second` and `result` types and initializations to `const float* first = GetRaw( firstHandle )` .

As a result there will be only one call to `GetRaw` instead of `batchSize` calls